script/Ciel/Day 5/05_01_CIEL05_1DASH.txt

```diff
@@ -348,6 +348,7 @@ And then, at that moment―――
 [sha:09d41ac2c24741a8a07b19bc79f4a74c07c762fe]{
 -- Page 48, Offset 10987.
 -- 「ああ？　どこの誰だよテメエ、なーに勝手にヒトん家にあがってるんですかぁー？」
+// group pr check: fix the voices of these two
```

Comment:
this is a Future Nice to Have during our later editing phase. Haru and Toshi sound soooo bland